tensorboard/plugins/hparams/api.py

```diff
@@ -0,0 +1,378 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Experimental public APIs for the HParams plugin.
+
+These are porcelain on top of `api_pb2` (`api.proto`) and `summary.py`.
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import abc
+import time
+
+import six
+
+from tensorboard.plugins.hparams import api_pb2
+from tensorboard.plugins.hparams import summary
+
+
+class Experiment(object):
+  """A top-level experiment description.
+
+  An experiment has a fixed set of hyperparameters and metrics, and
+  consists of multiple sessions. Each session has different associated
```

Comment:
Here, is it okay if I interpret `session` as `tf.Session`? Or does it map better to TB's `run`s?